tests/integration/standard/test_metrics.py

```diff
@@ -143,6 +143,8 @@ def test_unavailable(self):
         self.assertTrue(results)
 
         # Stop node gracefully
+        # Sometimes this commands continues with the other nodes having not noticed
+        # 1 is down, and a Timeout error is returned instead of an Unavailable
```

Comment:
Is this just a note for future triage?

Also, if `stop(wait=True, wait_other_notice=True)` returns without others noticing, that sounds like a CCM bug?